chia/util/merkle_set.py

```diff
@@ -120,18 +120,17 @@ def _audit(self, hashes: List[bytes], bits: List[int]) -> None:
 class MerkleSet:
     root: Node
 
-    def __init__(self, root: Optional[Node] = None):
-        if root is None:
-            self.root = _empty
+    def __init__(self, leafs: List[bytes32], *, set_root: Optional[Node] = None):
```

Comment:
Any need to be more specific than this?  If changing this then other places can just pass generators without adding `[]` to them to create lists via comprehensions.

```suggestion
    def __init__(self, leafs: Iterable[bytes32], *, set_root: Optional[Node] = None):
```